src/main/scala/carldata/series/Outliers.scala

```diff
@@ -0,0 +1,39 @@
+package carldata.series
+
+object Outliers {
+
+  /** Remove outliers */
+  def removeOutliers[V: Numeric](xs: TimeSeries[V], min: V, max: V)(implicit num: Numeric[V]): TimeSeries[V] = {
+    val ys = xs.index.zip(xs.values).flatMap(x => {
+      if (num.compare(x._2, min) < 0) None
+      else if (num.compare(x._2, max) > 0) None
+      else Some(x)
+    }).unzip
+
+    TimeSeries(ys._1, ys._2)
+  }
+
+  /** Remove outliers by set min/max values on their place */
+  def trimOutliers[V: Numeric](xs: TimeSeries[V], min: V, max: V)(implicit num: Numeric[V]): TimeSeries[V] = {
+    val vs = xs.values.map(x => {
+      if (num.compare(x, min) < 0) min
+      else if (num.compare(x, max) > 0) max
+      else x
+    })
+    TimeSeries(xs.idx, vs)
+  }
+}
+
+case class Outliers[V](){
```

Comment:
Why there is a case class?

Reply:
My mistake, I wanted here ```class```. Class because I cannot put interpolate inside Object, met:
```
found   : (Float, Float) => Float
[error]  required: (AnyVal, AnyVal) => AnyVal
```

Reply:
Also, from methods name I will trim suffix, so ```interpolateOutliers``` will be renamed to ```interpolate```

Reply:
This is very good idea

Reply:
Regarding the class. Let me look at it. I don't like to require class here

Reply:
This definition should be:
```
def interpolate[V: Numeric](xs: TimeSeries[V], min: V, max: V, f: (V, V) => V)(implicit num: Numeric[V]): TimeSeries[V]
```

Also fix for the tests:
```
Outliers.interpolate(series, 1f, 10f, f) shouldBe expected
```

The  parameters should have the same type as TimeSeries